fish-rust/src/abbrs.rs

```diff
@@ -0,0 +1,479 @@
+#![allow(clippy::extra_unused_lifetimes, clippy::needless_lifetimes)]
+use std::{
+    collections::HashSet,
+    sync::{Arc, Mutex, MutexGuard},
+};
+
+use crate::{
+    wchar::L,
+    wchar_ffi::{WCharFromFFI, WCharToFFI},
+};
+use cxx::{CxxWString, UniquePtr};
+use once_cell::sync::Lazy;
+pub use widestring::{Utf32Str as wstr, Utf32String as WString};
```

Comment:
No need to do the import dance here again.
Make it `use crate::wchar::{wstr, WString}`.